### DISCUSSION TITLE
Implementing Anti Captcha - Captcha Solving Service

### DISCUSSION THREAD

**Discussion by undauntedjogging** [jef/streetmerchant#1642] [General]
has anyone a solution to implement a service like anti captcha such as https://anti-captcha.com/mainpage to solve the captchas automatically by api?

at the moment i have a lot of problems because i am getting captchas on the site.
Has anyone a good solution for that?

**Comment by undauntedjogging** [comment]
it will work with the chrome extension from the service.
How is that possible to use Chrome with PUPPETEER_EXECUTABLE_PATH and extension?

If chrome opens i get every time the message in headless mode "organised by your company" and it snot able to activate chrome extensions.

**Reply by wofnull** [reply]
Chrome extensions will not work, since the streetmerchant script spawns an own instance of chromium per install in incognito mode, where no login into any google account is possible and the use of addins is disabled.

**Reply by undauntedjogging** [reply]
Yes I recognised the same. 
Any other possibilities to get the default chrome profile without „testing or company“ mode ?!
